src/main/java/edu/kis/powp/jobs2d/drivers/adapter/AdapterImpl.java

```diff
@@ -4,15 +4,18 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
 
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class AdapterImpl extends DrawPanelController implements Job2dDriver {
+	private final DrawerFeature drawerFeature;
 	private int startX = 0, startY = 0;
 
-	public MyAdapter() {
+	public AdapterImpl() {
 		super();
+		this.drawerFeature = new DrawerFeature();
```

Comment:
Direct coupling with DrawerFeature class is unnecessary.
Creating new instance of DrawerFeature is also unnecessary (static public members)